tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature

```diff
@@ -146,3 +146,30 @@ Feature: Share a file or folder that is inside a space
     Then the HTTP status code should be "200"
     And the information about the last share for user "Brian" should include
       | expiration |  |
+
+
+  Scenario: check the end of expiration date in user share
+    Given user "Alice" has created a share inside of space "share sub-item" with settings:
+      | path       | folder                   |
+      | shareWith  | Brian                    |
+      | role       | viewer                   |
+      | expireDate | 2042-01-01T23:59:59+0100 |
+    And user "Brian" has accepted share "/folder" offered by user "Alice"
+    When user "Alice" expires the last share
+    Then the HTTP status code should be "200"
+    Then as "Brian" folder "Shares/folder" should not exist
+
+  @issue-5823 @skip
+  Scenario: check the end of expiration date in group share
```

Comment:
This is skipped because of this issue https://github.com/owncloud/ocis/issues/5823